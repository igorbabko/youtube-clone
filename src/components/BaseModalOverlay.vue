<template>
  <div class="fixed inset-0 bg-black bg-opacity-80"></div>
</template>
