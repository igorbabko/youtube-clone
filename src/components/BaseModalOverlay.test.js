import { render } from '@testing-library/vue'
import BaseModalOverlay from './BaseModalOverlay.vue'

it('renders correctly', () => {
  const { html } = render(BaseModalOverlay)
  const comment = 'A very very long comment.'
  const video = {
    id: Math.ceil(Math.random() * 100),
    title: 'Video Title',
    publishedAt: new Date()
  }

  expect(html()).toMatchSnapshot()
  expect(comment).toMatchSnapshot()
  expect(video).toMatchSnapshot({
    id: expect.any(Number),
    publishedAt: expect.any(Date)
  })
})
