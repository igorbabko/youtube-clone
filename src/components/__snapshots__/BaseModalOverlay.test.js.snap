// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div class=\\"fixed inset-0 bg-black bg-opacity-80\\" data-testid=\\"base-modal-overlay\\"></div>"`;

exports[`renders correctly 2`] = `"A very very long comment."`;

exports[`renders correctly 3`] = `
Object {
  "id": Any<Number>,
  "publishedAt": Any<Date>,
  "title": "Video Title",
}
`;
